client/app/components/app-header/app-header.html

```diff
@@ -1,5 +1,5 @@
-<nav class="navbar navbar-default app-header" role="navigation">
-  <div class="container"">
+<nav class="navbar navbar-default navbar-fixed-top app-header" role="navigation">
```

Comment:
Actually we changed it to be not-fixed on purpose. 

I think it's still fixed on shared dashboards -- that's a mistake.

Reply:
My bad - I had to ask which is right. Will fix now